contrib/kube-prometheus/jsonnet/kube-prometheus/kube-prometheus-thanos.libsonnet

```diff
@@ -138,5 +138,44 @@ local servicePort = k.core.v1.service.mixin.spec.portsType;
       statefulSet.mixin.spec.template.spec.withVolumes([
         volume.fromEmptyDir('data'),
       ]),
+
+    thanosCompactorStatefulset:
+      local statefulSet = k.apps.v1beta2.statefulSet;
+      local volume = statefulSet.mixin.spec.template.spec.volumesType;
+      local container = statefulSet.mixin.spec.template.spec.containersType;
+      local containerEnv = container.envType;
+      local containerVolumeMount = container.volumeMountsType;
+
+      local labels = { app: 'thanos', 'thanos-peers': 'true' };
+
+      local c =
+        container.new('thanos-compactor', $._config.imageRepos.thanos + ':' + $._config.versions.thanos) +
+        container.withArgs([
+          'compact',
+          '--log.level=debug',
```

Comment:
Are we sure debug is what all user wants? Might be bit spammy?

Reply:
I think I'd prefer to have this be done on an individual basis, and not have the `--log.level` flag at all by default. Feel free to have the removal from all components be a follow up though.